sycl/doc/Array_Kernel_Parameters.md

```diff
@@ -0,0 +1,435 @@
+<h2>Array Parameters of SYCL Kernels</h2>
+
+<h3>Introduction</h3>
+
+This document describes the changes to support passing arrays to SYCL kernels 
```

Comment:
changes to what? Please add

Reply:
This document not only describes arrays of parameters, but mechanisms behind kernel parameter generation in general. So I suggest to rename the document to reflect that. Arrays is just one of the topics covered here.

Would be great to add a link from CompilerAndRuntimeDesign.md to this document,